lib/registration.pm

```diff
@@ -374,9 +374,7 @@ sub skip_package_hub_if_necessary {
     my ($addon) = @_;
     my $skip_package_hub = 0;
     if (is_sle('15-SP2+') && $addon eq 'phub') {
-        if (check_var('FLAVOR', 'Online')) {
-            record_soft_failure('bsc#1151373 - Missing or broken repository after registering module PackageHub');
```

Comment:
Unfortunately, I guess we need to do more work here. Your VR works only because we still keep skipping package hub module. So we need at least similar message to Full medium explaining why it doesn't work.
As an alternative, we can disable this module from the test suite we have in our job group.

Reply:
As discussed with @foursixnine for full medium stays the skip with `record_info` as phub is not an online product. For Online medium there was a false positive, so phub shouldn't be rewriten by the proxy, because there is a different process of release comparing with other modules. Verification enabling phub for Online medium updated in the first comment.

Reply:
Right, but now it fails. Is it a valid bug or setup issue?

Reply:
It fails installing patterns: http://rivera-workstation.suse.cz/tests/1226#step/select_patterns/10
but in a package that comes from transaction module, which is weird, https://scc.suse.com/packages?name=SUSE%20Linux%20Enterprise%20Server&version=15.2&arch=x86_64&query=health-checker-plugins-kubic&module=
Retriggering current job to see if is really phub the one introducing this: https://openqa.suse.de/tests/4274266

Reply:
yes, so indeed phub enabling is creating this conflict, because we select more patterns, @rwx788 does it make sense to file the bug here? also @foursixnine wdyt?

Reply:
So, the source of the problem is `evas-generic-loader` that comes from [phub](https://scc.suse.com/packages?name=SUSE%20Linux%20Enterprise%20Server&version=15.2&arch=x86_64&query=evas-generic-loader&package_id=19862054) so in deed, you do need that module apparently, if all patterns is to be tested. 

But tbh, looks like a product bug. As: Having to install a package from the backports to install something from full medium, defeats the purpose, doesn't it?

Reply:
+1, it's a product issue which we should report as released SLES will have this issue too.

Reply:
Bug filed: https://bugzilla.suse.com/show_bug.cgi?id=1172074